allvars.h

```diff
@@ -520,7 +520,7 @@ extern struct particle_data
     pthread_spinlock_t SpinLock;
 #endif
 
-    float GravCost;     /*!< weight factor used for balancing the work-load */
+    int GravCost;     /*!< weight factor used for balancing the work-load */
```

Comment:
True. I wonder though if we should down-weight the high redshift hydro. It seems wrong...hydro is expensive at high redshift.